huxley/api/serializers/delegate.py

```diff
@@ -25,6 +31,32 @@ class Meta:
                   'session_three',
                   'session_four', )
 
+    def update(self, instance, validated_data):
+        if ('assignment' in validated_data and
+                validated_data['assignment'] != None and
+                not instance.assignment):
+
+            names = instance.name.split(' ')
+            username = names[0] + '_' + str(instance.id)
+            password = BaseUserManager().make_random_password(10)
+            user = User.objects.create(delegate=instance, username=username,\
+                                       user_type=User.TYPE_DELEGATE,\
+                                       last_login=datetime.now(),
```

Comment:
Is this a field we added? If it's default Django, wouldn't this be set by default?

Reply:
It's something we set when creating accounts for advisors; I wasn't sure of the motivation there so I figured it wouldn't hurt to include it here as well.